src/test/java/org/jabref/logic/importer/fetcher/GoogleScholarTest.java

```diff
@@ -27,69 +27,69 @@
 @FetcherTest
 class GoogleScholarTest implements SearchBasedFetcherCapabilityTest, PagedSearchFetcherTest {
 
-    private GoogleScholar finder;
-    private BibEntry entry;
+    private GoogleScholar fetcher;
 
     @BeforeEach
     void setUp() {
         ImportFormatPreferences importFormatPreferences = mock(ImportFormatPreferences.class);
         when(importFormatPreferences.getFieldContentFormatterPreferences()).thenReturn(
                 mock(FieldContentFormatterPreferences.class));
-        finder = new GoogleScholar(importFormatPreferences);
-        entry = new BibEntry();
+        fetcher = new GoogleScholar(importFormatPreferences);
     }
 
     @Test
     @DisabledOnCIServer("CI server is blocked by Google")
     void linkFound() throws IOException, FetcherException {
-        entry.setField(StandardField.TITLE, "Towards Application Portability in Platform as a Service");
+        BibEntry entry = new BibEntry()
+                .withField(StandardField.TITLE, "Towards Application Portability in Platform as a Service");
 
         assertEquals(
                 Optional.of(new URL("https://www.uni-bamberg.de/fileadmin/uni/fakultaeten/wiai_lehrstuehle/praktische_informatik/Dateien/Publikationen/sose14-towards-application-portability-in-paas.pdf")),
-                finder.findFullText(entry)
+                fetcher.findFullText(entry)
         );
     }
 
     @Test
     @DisabledOnCIServer("CI server is blocked by Google")
     void noLinkFound() throws IOException, FetcherException {
-        entry.setField(StandardField.TITLE, "Curriculum programme of career-oriented java specialty guided by principles of software engineering");
+        BibEntry entry = new BibEntry()
+                .withField(StandardField.TITLE, "Curriculum programme of career-oriented java specialty guided by principles of software engineering");
 
-        assertEquals(Optional.empty(), finder.findFullText(entry));
+        assertEquals(Optional.empty(), fetcher.findFullText(entry));
     }
 
     @Test
     @DisabledOnCIServer("CI server is blocked by Google")
     void findSingleEntry() throws FetcherException {
-        entry.setType(StandardEntryType.InProceedings);
-        entry.setCitationKey("geiger2013detecting");
-        entry.setField(StandardField.TITLE, "Detecting Interoperability and Correctness Issues in BPMN 2.0 Process Models.");
-        entry.setField(StandardField.AUTHOR, "Geiger, Matthias and Wirtz, Guido");
-        entry.setField(StandardField.BOOKTITLE, "ZEUS");
-        entry.setField(StandardField.YEAR, "2013");
-        entry.setField(StandardField.PAGES, "41--44");
+        BibEntry entry = new BibEntry(StandardEntryType.InProceedings)
+                .withCitationKey("geiger2013detecting")
```

Comment:
There is nothing wrong with the `set...` methods. The `with...` methods were added to quickly add one or two field values.